CODE_OF_CONDUCT.md

```diff
@@ -1,29 +1,30 @@
-# 📰 CODE OF CONDUCT
-   
+# 📰 Code of Conduct
+
 <!-- 
-! Joshith, make all these points as numbered lists
+TODO: Joshith, make all these points as numbered lists
```

Comment:
That exclamation sign is for the use of the Better Comments extension in VS Code. Better Comments extension allows us to view comments in a more colorful way.